Sources/xcodeproj/Objects/Project/PBXProj.swift

```diff
@@ -70,10 +70,21 @@ public final class PBXProj: Decodable {
         classes = try container.decodeIfPresent([String: Any].self, forKey: .classes) ?? [:]
         let objectsDictionary: [String: Any] = try container.decodeIfPresent([String: Any].self, forKey: .objects) ?? [:]
         let objectsDictionaries: [String: [String: Any]] = (objectsDictionary as? [String: [String: Any]]) ?? [:]
-        try objectsDictionaries.forEach { reference, dictionary in
-            let object = try PBXObject.parse(reference: reference, dictionary: dictionary, userInfo: decoder.userInfo)
+        
+        let parser = PBXObjectParser(
+            userInfo: decoder.userInfo
+        )
+        let dictionary = NSDictionary(dictionary: objectsDictionaries)
+        dictionary.enumerateKeysAndObjects(options: .concurrent) { (key, obj, _) in
+            let reference = key as! String
+            let dictionary = obj as! Dictionary<String, Any>
+            guard let object = try? parser.parse(
```

Comment:
I'd not use `try?` here and bubble up the error instead. By doing that we prevent the error from going unnoticed and help us detect if there's something to be supported/fixed.